daffodil-propgen/src/main/scala/org/apache/daffodil/propGen/TunableGenerator.scala

```diff
@@ -76,21 +76,25 @@ class TunableGenerator(schemaRootConfig: scala.xml.Node, schemaRootExt: scala.xm
     |      if (configOpt.isDefined) {
     |        val loader = new DaffodilXMLLoader()
     |        val node = loader.load(URISchemaSource(configOpt.get), Some(XMLUtils.dafextURI))
-    |        val trimmed = scala.xml.Utility.trim(node)
-    |        val tunablesNode = (trimmed \ "tunables").headOption
-    |        val tunablesMap: Map[String, String] = tunablesNode match {
-    |          case None => Map.empty
-    |          case Some(tunableNode) => {
-    |            tunableNode.child.map { n => (n.label, n.text) }.toMap
-    |          }
-    |        }
-    |        tunablesMap
+    |        tunablesMap(node)
     |      } else {
     |        Map.empty
     |      }
     |
     |    new DaffodilTunables().setTunables(configTunables)
     |  }
+    |
+    |  def tunablesMap(node: scala.xml.Node) = {
```

Comment:
What's our standard for including the type of the return value in a function definition? I personally prefer it since I can immediately know the return value, but maybe that' s not very Scala-y?

Reply:
It's public method, so I think should have the return type.